IoT/wled.1m.js

```diff
@@ -0,0 +1,102 @@
+#!/usr/local/bin/node
+//  -*- coding: utf-8 -*-xbar
+//  <xbar.title>WLED</xbar.title>
+//  <xbar.version>v1.0</xbar.version>
+//  <xbar.author>Billy Jacoby</xbar.author>
+//  <xbar.author.github>billyjacoby</xbar.author.github>
+//  <xbar.desc>Show and manage WLED devices from the menu bar.</xbar.desc>
+//  <xbar.dependencies>node,wled,axios</xbar.dependencies>
```

Comment:
Please add an image of what the plugin looks like when installed.

Reply:
Fixed in most recent commit!